unlock-app/src/middlewares/walletMiddleware.js

```diff
@@ -26,7 +26,7 @@ import {
 } from '../errors'
 import { TransactionType } from '../unlockTypes'
 import { hideForm } from '../actions/lockFormVisibility'
-import { transactionTypeMapping } from '../utils/types' // TODO change POLLING_INTERVAL into ACCOUNT_POLLING_INTERVAL
```

Comment:
This comment was in the wrong place. I have renamed `POLLING_INTERVAL` to `ACCOUNT_POLLING_INTERVAL` and updated all sites where it's used (this middleware and the associated test).